apps/web-app/app/routes/profile.tsx

```
@@ -53,6 +68,47 @@ export default function Profile() {
+            {permissions.permissions.canVerifyRoles && <li>✓ Verify roles</li>}
+          </ul>
+        </div>
+      )}
+
+      <div className="role-management">
+        <h2>Role Management</h2>
+        <button 
+          className="button primary" 
+          onClick={() => setShowUpgradeForm(!showUpgradeForm)}
+        >
+          {showUpgradeForm ? 'Cancel' : 'Request Role Upgrade'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)